lua/entities/ent_jack_aidfuel_kerosene/init.lua

```diff
@@ -2,20 +2,6 @@ AddCSLuaFile( "cl_init.lua" )
 AddCSLuaFile( "shared.lua" )
 include( "shared.lua" )
 
-function ENT:SpawnFunction( ply, tr )
-    local SpawnPos = tr.HitPos + tr.HitNormal * 16
-    local ent = ents.Create( "ent_jack_aidfuel_kerosene" )
-    ent:SetPos( SpawnPos )
-    ent:SetNWEntity( "Owner", ply )
-    ent:Spawn()
-    ent:Activate()
-    local effectdata = EffectData()
-    effectdata:SetEntity( ent )
-    util.Effect( "propspawn", effectdata )
-
-    return ent
-end

```

Comment:
Doesn't work, or just not needed?

Reply:
Not needed, gmod will pretty much just do this for you.